src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/iOS/iOSTokenCacheAccessor.cs

```diff
@@ -171,85 +171,29 @@ public void SaveAccount(MsalAccountCacheItem item)
 
             var type = AuthorityTypeToAttrType[item.AuthorityType];
 
-            var value = JsonHelper.SerializeToJson(item);
+            var value = item.ToJsonString();
 
             Save(account, service, generic, type, value);
         }
 
-        public void DeleteAccessToken(MsalAccessTokenCacheKey cacheKey)
+        public ICollection<MsalAccessTokenCacheItem> GetAllAccessTokens()
         {
-            var account = cacheKey.GetiOSAccountKey();
```

Comment:
>GetiOSAccountKey [](start = 35, length = 16)

are these GetiOS____Key() methods okay to remove?

Reply:
None of these methods are called from anywhere within the ADAL codebase.